blog/_posts/2018-07-09-introducing-TeamBarcelonaDuo.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Introducing Team Barcelona Duo
+layout: post
+created_at: Mon July 09 2018
+permalink: blog/introducing-TeamBarcelonaDuo
+current: blog
+author: Lori and Amalia
+twitter: HarlemExpat
+categories:
+- student-posts
+---
+Hello everyone, we are Lori and Amalia, two amazing ex-sociologist turned programmers. We are Team BarcelonaDuo. We met at [codebar](https://codebar.io/) <br> and are currently organizers there.
+
```

Comment:
Is there any reason for using <br> tag here? It seems unnecessary to me and breaks the sentence.